cli/cli.go

```diff
@@ -113,6 +124,41 @@ func Run() {
 	}
 }
 
+func commandExecutionInfo(args []string) (string, string, []string) {
```

Comment:
Can we have some tests for this logic? 👀 

I see that it correctly filters out flag _values_ (which could be sensitive), but we should also enforce this to prevent a future regression

Reply:
Great tip. I have added tests for this part.